common/patches/uf.cpp

```diff
@@ -1189,14 +1189,14 @@ namespace UF
 
 		unsigned char *__emu_buffer = in->pBuffer;
 		char *InBuffer = (char *)__emu_buffer;
-		uint32 HighestGuildID = 0;
+		uint32 actual_no_guilds = 0;
```

Comment:
`guild_count`? Naming seems a bit obtuse.

Reply:
Just above there is a variable called NumberOfGuilds, so I decided to use ActualNoOfGuilds, then converted to the standard so landed on actual_no_guilds.

Reply:
Gotcha. `actual_no_guilds` just seemed weird at first.